### PR TITLE
feat(fmt): add functionality from prisma-fmt as quick fixes

### PR DESCRIPTION
**Pull request by Druue**
closes https://github.com/prisma/team-orm/issues/1195

### CODE REVIEW

File: prisma-fmt/tests/code_actions/scenarios/add_missing_relation_attribute_args_fields/result.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+              "start": {
+                "line": 2,
+                "character": 19
+              },
+              "end": {
+                "line": 2,
+                "character": 51
+              }
+            },
+            "newText": "@relation(fields: [Postpost_id], references: [post_id])"
```
Comment (SevInf) [inline prisma-fmt/tests/code_actions/scenarios/add_missing_relation_attribute_args_fields/result.json:35]: `Postpost_id` does not seem right

File: prisma-fmt/src/code_actions/field.rs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        "is missing an opposite relation field on the model",
+    );
+
+    if diagnostics.is_empty() {
+        return;
+    }
+
+    // * We know that this is safe to unwrap here as otherwise this diagnostic would be
+    // * replaced with the one that we check for in `fn create_missing_block_for_model`
+    let target_model = context.db.find_model(target_name).unwrap();
```
Comment (SevInf) [inline prisma-fmt/src/code_actions/field.rs:33]:
```suggestion
    let Some(target_model) = context.db.find_model(target_name) else {
      return;
    }
```

Just to be safe

File: prisma-fmt/src/code_actions/field.rs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    let target_file_id = target_model.file_id();
+    let target_file_content = context.db.source(target_file_id);
+
+    target_model.fields().last().unwrap();
```
Comment (SevInf) [inline prisma-fmt/src/code_actions/field.rs:38]: Why are we doing this?

Comment (Druue) [reply prisma-fmt/src/code_actions/field.rs:38]: Which part specifically? @SevInf

Comment (SevInf) [reply prisma-fmt/src/code_actions/field.rs:38]: Only the line itself.  Seems like it's not doing anything except panicing if fields are empty?

Comment (Druue) [reply prisma-fmt/src/code_actions/field.rs:38]: Ah, no I see what you mean now, I think I was using it at one point and then didn't need it. Forgot to remove 😅

**Pull request merged**